wpimath/src/main/native/include/frc/controller/ArmFeedforward.h

```diff
@@ -88,11 +99,40 @@ class WPILIB_DLLEXPORT ArmFeedforward {
    * @param dt Time between velocity setpoints in seconds.
    * @return The computed feedforward in volts.
    */
+  [[deprecated("Use the current/next velocity overload instead.")]]
   units::volt_t Calculate(units::unit_t<Angle> currentAngle,
                           units::unit_t<Velocity> currentVelocity,
                           units::unit_t<Velocity> nextVelocity,
                           units::second_t dt) const;
 
+  /**
+   * Calculates the feedforward from the gains and setpoints.
+   *
+   * @param currentAngle The current angle. This angle should be measured from
+   * the horizontal (i.e. if the provided angle is 0, the arm should be parallel
+   * to the floor). If your encoder does not follow this convention, an offset
+   * should be added.
+   * @param setpoint The current velocity setpoint.
+   * @return The computed feedforward in volts.
+   */
+  units::volt_t Calculate(units::unit_t<Angle> currentAngle,
+                          units::unit_t<Velocity> setpoint) const;
```

Comment:
What is this overload for? I don't get why this is called setpoint.

Reply:
I adjusted the doc

It should now read

```
   * Calculates the feedforward from the gains and setpoint assuming discrete
   * control. Use this method when the setpoint does not change.
```

Reply:
That's not my point. `setpoint` doesn't make sense in this context. The arm state include angle and angular velocity, so you'd have an angle setpoint and angular velocity setpoint. We don't include the word setpoint in feeedforward argument names because it's redundant.